### PR TITLE
Proofread readme

### PR DESCRIPTION
**Pull request by greg0ire**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,15 +20,16 @@ Documentation
...
 
-**1.**  Add dependency with composer
+**1.**  Add dependency with Composer
 
 ```bash
 composer require kunicmarko/sonata-auto-configure-bundle
 ```
 
-**2.** Register the bundle in your `bundles.php`
+**2.** Register the bundle in the kernel of your application:
```
Comment (kunicmarko20) [inline README.md]: why would you say kernel here? since it should be added in the bundles file?

Comment (greg0ire) [reply README.md]: :thinking: I see how that wording is confusing, let me rephrase, although I think this effectively is what happens here:
```
    // src/Kernel.php

    public function registerBundles()
    {
        $contents = require $this->getProjectDir().'/config/bundles.php';
        foreach ($contents as $class => $envs) {
            if ($envs[$this->environment] ?? $envs['all'] ?? false) {
                yield new $class();
            }
        }
    }

```

Comment (kunicmarko20) [reply README.md]: ye, I know, but not sure most of the other people know that that is why I wrote `bundles.php`

**Review by kunicmarko20 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (kunicmarko20) [reply README.md]: looks good

**Comment by kunicmarko20** [comment]
Thanks @greg0ire !
